src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/0.1.0/Generated.PowerShell.Commands/SwaggerPathCommands/Get-ComputeFabricOperation.ps1

```diff
@@ -0,0 +1,135 @@
+<#
+The MIT License (MIT)
+
+Copyright (c) 2017 Microsoft
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+#>
+
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath .. | Join-Path -ChildPath .. | Join-Path -ChildPath "GeneratedHelpers.psm1")
+<#
+.DESCRIPTION
+    Get the status of a compute fabric operation.
+
+.PARAMETER ComputeOperationResult
+    Id of a compute fabric operation.
+
+.PARAMETER Provider
+    Name of the provider.
+
+.PARAMETER Location
+    Location of the resource.
+
+.EXAMPLE
+
+Get-AzsComputeFabricOperation -Location "local" -Provider "Microsoft.Fabric.Admin" -ComputeOperationResult "fdcdefb6-6fd0-402c-8b0c-5765b8fc4dc1"
+
+ProvisioningState
+-----------------------
+Succeeded
+
+#>
+function Get-ComputeFabricOperation
+{
+    [OutputType([Microsoft.AzureStack.Management.Fabric.Admin.Models.OperationStatus])]
+    [CmdletBinding(DefaultParameterSetName='ComputeFabricOperations_Get')]
+    param(    
+        [Parameter(Mandatory = $true, ParameterSetName = 'ComputeFabricOperations_Get')]
+        [System.String]
+        $ComputeOperationResult,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'ComputeFabricOperations_Get')]
+        [System.String]
+        $Provider,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'ComputeFabricOperations_Get')]
+        [System.String]
+        $Location
+    )
+
+    Begin 
+    {
+	    Initialize-PSSwaggerDependencies -Azure
```

Comment:
What does this do, exactly?

Reply:
It is possible that for long running operations you can get the operationId that allows you to check the status of the operation.